src/App.tsx

```diff
@@ -1,9 +1,58 @@
 import './App.scss';
+import { useEffect, useState } from 'react';
+import { NavLink, Route, Routes } from 'react-router-dom';
+import 'bulma/css/bulma.css';
 
-const App = () => (
-  <div className="App">
-    <h1>People table</h1>
-  </div>
-);
+import { getPeople } from './api';
+import { PeopleTable } from './components/PeopleTable/PeopleTable';
+import { Loader } from './components/Loader';
 
-export default App;
+export const App:React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+
+  const activeLink = ({ isActive }: any) => (isActive
+    ? 'title is-1 has-text-primary'
+    : 'title is-1 has-text-white');
+
+  useEffect(() => {
+    getPeople().then(data => setPeople(data));
+  }, []);
```

Comment:
it may be better to do this in PeoplePage component: 
"8. Create getPeople method fetching people from [API](https://mate-academy.github.io/react_people-table/api/people.json) when PeoplePage is opened"

Reply:
I wrote it in a different way